restapi/src/main/resources/swagger-ui-cust/index.html

```diff
@@ -56,7 +53,26 @@
             return tokenValue == null ? "" : tokenValue.trim();
           }
         }
-      })
+      };
+
+      const pathname = window.location.pathname;
+      if (pathname !== '/swagger-ui') {
```

Comment:
Could we support proxied URLs without an `if` statement? Could we make the logic to work with empty prefixes?

I believe it would be more robust to avoid forks in the processing logic. WDYT?

Reply:
We can remove that check, but then we would register a request interceptor that does nothing.

Reply:
that may still happen if line 68 is executed without a proxy? :thinking:

Reply:
If there is no proxy, `/swagger-ui` should be exposed at the root, and then we don't enter the if block and don't install the request interceptor.

Reply:
Ah, TIL :)